tests/containers/bci_prepare.pm

```diff
@@ -98,15 +99,17 @@ sub run {
     }
 
     # For BCI tests using podman, buildah package is also needed
-    install_buildah_when_needed($host_distri) if ($engine eq 'podman');
+    install_buildah_when_needed($host_distri) if ($engines =~ /podman/);
 
     record_info('Clone', "Clone BCI tests repository: $bci_tests_repo");
     my $branch = $bci_tests_branch ? "-b $bci_tests_branch" : '';
-    assert_script_run("git clone $branch -q --depth 1 $bci_tests_repo");
+    assert_script_run("git clone $branch -q --depth 1 $bci_tests_repo /root/BCI-tests");
 
     # Pull the image in advance
     if (my $image = get_var('CONTAINER_IMAGE_TO_TEST')) {
         record_info('IMAGE', $image);
+        # If $engines are multiple (e.g. CONTAINER_RUNTIME=podman,docker), we just pick one of them for this check
+        my $engine = ($engines =~ /podman/) ? 'podman' : 'docker';
```

Comment:
Would be better?
```
if ($engine =~ /podman/) { ... }
else if($engine =~ /docker/) {...}
else No valid engine

Reply:
ok, why not

Reply:
if **$engines**="podman,docker", where will it span both values in order to repeat the test for each engine?

Reply:
I have updated it. This module is executed only 1 time, I just pick one of the engines that are in the variable.

Reply:
But _docker_ set the pulled repo in a different place than _podman_, so we should repeat the $engine pull for each type

Reply:
Right, but this code block is just to verify the build tag of the image and report also the size to some database. The BCI-tests will pull the image themselves when needed.